app/views/doc/credential_helpers.html.erb

```diff
@@ -0,0 +1,47 @@
+<%- @section    = 'documentation' %>
+<%- @subsection = 'credential-helpers' %>
+<%- @page_title = 'Git credential helpers' %>
+
+<div id="main">
+  <h1>Git credential helpers</h1>
+
+  <p>This page lists available <a href="https://git-scm.com/docs/gitcredentials">credential helpers</a>.</p>
+
+  <h2>Included in Git</h2>
+
+  <ul>
+    <li><a href="https://git-scm.com/docs/git-credential-cache">git-credential-store</a>: saves credentials in plaintext.</li>
+    <li><a href="https://git-scm.com/docs/git-credential-cache">git-credential-cache</a>: holds credentials temporarily in process memory. <a href="https://github.com/git-for-windows/git/issues/3892">Not available on Windows</a>. (NB. Since credentials are lost when the cache expires, this is inconvenient to store personal access tokens.)</li>
+  </ul>
+
+  <h2>Platform specific storage</h2>
+  <p></p>
+  <ul>
+    <li>git-credential-wincred: included with Git for Windows.</li>
+    <li>git-credential-osxkeychain: stores in macOS keychain.</li>
```

Comment:
Google didn't yield a ton of results for documentation on this one either - perhaps we could use https://git-scm.com/book/sv/v2/Git-Tools-Credential-Storage?

Reply:
Added link about macOS keychain app.